src/components/Profile/Profile.module.css

```diff
@@ -0,0 +1,67 @@
+.main-div {
+    width: 1100px;
```

Comment:
Would it look good if you used %s instead of px for container width?

Reply:
Great idea! I will fix it.